integrations/access/accessrequest/app.go

```diff
@@ -234,12 +234,20 @@ func (a *App) onPendingRequest(ctx context.Context, req types.AccessRequest) err
 		return trace.Wrap(err)
 	}
 
+	loginsByRole, err := a.getLoginsByRole(ctx, req)
+	if trace.IsAccessDenied(err) {
+		log.Warnf("Missing permissions to get logins by role. Please add role.read to the associated role. error: %s", err)
```

Comment:
@bernardjkim Did you verify that the absence of this permission keeps old behavior and does not break access request plugins?

Reply:
Yup, if the permissions are missing Teleport will just log the warning and use the old behavior.